project.clj

```diff
@@ -69,7 +69,8 @@
                  [puppetlabs/kitchensink ~ks-version]
                  [puppetlabs/trapperkeeper ~tk-version]
                  [puppetlabs/trapperkeeper-webserver-jetty9 ~tk-jetty9-version]
-                 [prismatic/schema "0.2.2"]
+                 [prismatic/schema "0.4.0"]
+                 [prismatic/plumbing "0.4.2"]
```

Comment:
Does TK and schema 0.4.0 now depend on different versions of plumbing? Is that why we need to declare a direct dependency on plumbing?